src/components/Modal/Modal.mdx

```diff
@@ -5,13 +5,24 @@ import { Primary } from './Modal.stories';
 
 <Canvas of={Primary} />
 
-## Example
+## Usage
 
-```html
-<LobButton @click="isModalVisible = true">Open Modal</LobButton>
+**Note:** You should **not** utilize both the prop and slot for a particular
+field (i.e. Don't use the `header` prop with the `header` slot) as this will
```

Comment:
how do other component libraries handle this? can we not add some `v-if="!prop"` to slots (or the other way round)?

Reply:
I was thinking about putting the prop value within the slot definition as the slot fallback. I thought it could be more confusing by conditionally choosing one to show and one to hide without some reason behind which gets hidden